.storybook/static/cmsgov-theme.css

```diff
@@ -511,7 +511,7 @@
 --typography-heading-lg__font-weight: 700;
 --typography-heading-md__font-weight: 700;
 --typography-heading-sm__font-weight: 400;
---typography-body__font-family: inherit;
+--typography-body__font-family: 'Open Sans', Helvetica, sans-serif;
```

Comment:
I updated these pre-existing `--typography-` vars to explicitly point to the theme's body font. As it was before, this var was relying on the `--font-sans` being set on `.ds-base`. Now the proper body fonts are being assigned to the `body` tag